docs/glossary.md

```diff
@@ -10,17 +10,23 @@ title: Glossary
 
 **[Ack](https://nymtech.net/docs/stable/architecture/traffic-flow/#acks)** - Short for "acknowledgement". Whenever a hop is completed, the recieving node will send back an 'ack', so that the sending node knows that the packet was recieved.
 
+**Attacker** - A person or entity who attempts to gain unauthorised access to a computer system, network, or other information resource.
+
+**Binary** - IS a type of computer file that contains machine code, which is a set of instructions that a computer can execute directly. The file consists of a series of 1s and 0s, which are the two digits used in the binary number system.
+
 **Blockchain** - A distributed database that maintains a continuously growing list of ordered records, called blocks.
 
 **[Coconut](https://nymtech.net/docs/stable/coconut/)** - A distributed cryptographic signing scheme providing a high degree of privacy for its users.
 
 **[CLI](https://nymtech.net/docs/stable/nym-cli/)** - A command-line interface (CLI) is a text-based user interface (UI) used to run programs, manage computer files and interact with the computer.
 
+**Credentials** - Unique values that prove the legitimacy of a user or individual.
+
 **Centralisation** - Refers to the concentration of control of an activity or organisation under a single authority, or small group of authorities.
 
-**Credentials** - Unique values that prove the legitimacy of a user or individual.
+**[CosWasm](https://cosmwasm.com)** - COSMOS WebAssembly is a WebAssembly (Wasm) interpreter that is used in the Cosmos SDK, a framework for building decentralised applications (dApps) on the Cosmos network.
 
-**[Crytography](https://en.wikipedia.org/wiki/Cryptography)** -  Is a practice and study of protecting information and communications through the use of codes, so that only those for whom the information is intended can read and process it.
+**DAO** - Stands for Decentralised Autonomous Organisation. It's a type of organisation that is ran through the use of smart contracts on a blockchain. Also it's a digital platform that operates on a set if rules encoded in a computer program, and is governed by the people who use it, rather than by a central authority.
```

Comment:
Sort grammar: 
* 'that is ran' -> 'run' 
* remove 'It's' @ start of 2nd sentence. 
* Arguably its an organisation _made up_ of smart contracts
* Remove 'Also its a digital platform that' -> 'It'